src/main/java/com/google/api/codegen/viewmodel/StaticLangApiView.java

```diff
@@ -61,6 +61,9 @@
 
   public abstract boolean hasLongRunningOperations();
 
+  @Nullable // Used in Java
+  public abstract String stubInterfaceName();
```

Comment:
I think it would be neat to have a custom variant of `AutoValue` that could default to a "not set" string for values that aren't set. That might be a path to yak shaving though...